nio/block/mixins/persistence/persistence.py

```diff
@@ -88,3 +104,22 @@ def stop(self):
         # Do one last save before stopping
         self._save()
         super().stop()
+
+    def serialize(self):
```

Comment:
These should be called `persistence_deserialize` and `persistence_serialize` to avoid name collisions in blocks.